ledger/acctupdates.go

```diff
@@ -1552,8 +1553,9 @@ func (au *accountUpdates) lookupWithRewardsImpl(rnd basics.Round, addr basics.Ad
 	}
 }
 
-// lookupWithoutRewardsImpl returns the account data for a given address at a given round.
-func (au *accountUpdates) lookupWithoutRewardsImpl(rnd basics.Round, addr basics.Address, syncronized bool) (data basics.AccountData, validThrough basics.Round, err error) {
+// lookupWithoutRewards returns the account data for a given address at a given round.
+// The resultant AccountData does not include application states.
```

Comment:
the line `The resultant AccountData does not include application states.` need to be removed.

Reply:
done